openshift/template.yaml

```diff
@@ -542,6 +823,22 @@ objects:
         targetPort: ${{SVC_TARGET_PORT}}
     selector:
       app: assisted-service
+
+- apiVersion: v1
+  kind: Service
+  metadata:
```

Comment:
do we need it? We don't expose anything from this pod, right?

Reply:
I think it isn't needed, I will remove it.

Reply:
Done.